vernac/vernacstate.mli

```diff
@@ -62,15 +62,23 @@ val unfreeze_interp_state : t -> unit
 (* WARNING: Do not use, it will go away in future releases *)
 val invalidate_cache : unit -> unit
 
-(* STM-specific state handling *)
+(** STM-specific state handling *)
 module Stm : sig
+
+  (** Proof state + meta/evar counters *)
   type pstate
 
-  (** Surgery on states related to proof state *)
   val pstate : t -> pstate
   val set_pstate : t -> pstate -> t
-  val non_pstate : t -> Summary.frozen * Lib.frozen
+
+  (** Rest of the state, unfortunately this is used in low-level so we need to expose it *)
+  type rstate = Summary.frozen * Lib.frozen
+  val rstate : t -> rstate
```

Comment:
I don't see why you are renaming this. `pstate` and `non_pstate` seems more informative to me than `pstate` and `rstate`.

Reply:
Was meant to be "rest of state", I've kept `non_pstate` .